src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -67,6 +67,7 @@ Commands
             check  Check your project for errors.
               dev  Start the development server.
              docs  Open documentation in your web browser.
+             info  Emit useful information about Astro current setup. Useful for debugging.
```

Comment:
```suggestion
             info  List info about your current Astro setup.
```
This is way longer than the others. Do you think something like this is good enough?

Reply:
No problem, I will update the code too